include/config.h

```diff
@@ -5,7 +5,7 @@
 #include <glib.h>
 
 #ifndef CONFIG_FILE
-#define CONFIG_FILE "/etc/lightdm/lightdm-minigreeter.conf"
+#define CONFIG_FILE "/etc/lightdm/lightdm-mini-greeter.conf"
```

Comment:
Hadn't noticed this has been wrong this entire time because this code is never run since autotools defines `CONFIG_FILE` as `$(sysconfdir)/lightdm/lightdm-mini-greeter.conf`.

Can you have cmake define the same macro so it can be dynamic(keep this fix as well).

Reply:
Honestly...i don't know, but i am sure something can be done.